src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
+
+   1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI**, click **Destinations**, and then select **ServiceNow**.
+
+   2. Enter the following information:
+
+        * **Destination Name:** Custom name to identify the destination.
 
-    2. Enter the following information:
+        * **Domain:** The URL of the destination.
 
-        * **Name**: Custom name to identify the destination.
+        * **User-name:** The name of the user.
 
-        * **API Key**: For this integration, you will be asked to provide a REST API Key. There're two types of REST API keys in PagerDuty, [general access](https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-general-access-rest-api-key) and [user tokens](https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-personal-rest-api-key).
+        * **Password:** The user's password.
 
 
   Before saving the destination, we recommend testing the connection by clicking the **Test connection** button.
 
-      <img
-        width="80%;"
-        title="PagerDuty account-level destination configuration"
-        alt="A screenshot of a PagerDuty account-level destination configuration"
-        src={accountsPagerDutyAccountDestinationNewUi}
-      />
 
 
-  ### Two-way sync [#two-way-pagerduty]
+  ### Two-way sync [#two-way-snow]
 
-  To enable two-way sync, turn on the **two-way integration** toggle.
+  To configure the two-way integration:
 
-  When turned on, a PagerDuty subscription is created at a later stage for the selected PagerDuty service (see [customize a message template](#message-pagerduty)). The webhook contains access details to New Relic (URL and New Relic API key).
+      1. Turn on the `two-way integration` toggle.
 
-  By default, any change of state to the PagerDuty incident created by New Relic will sync back to New Relic.
+      2. Open and download this <a href="https://storage.googleapis.com/newrelic-notifications/snow-two-way-sync/Newrelic-Snow-Sync-Configurations.xml" download="NewRelic-SNow-Sync-Configurations.xml">XML file</a>, which includes the business rule triggering events back to New Relic.
 
-  <Callout variant="important">
-    Please note that if you're a PagerDuty Event Intelligence or Digital Operations customer using Intelligent Alert Grouping on a given service, there could be a potential mismatch in PagerDuty Incidents sent back to New Relic.
-  </Callout>
+      3. In the ServiceNow sidebar menu, go to **System Definition > Business Rules**.
 
+      4. Click the menu icon in one of the column headers, select **Import XML**, and upload the XML file you downloaded.
 
-  ### Sync with New Relic workflows [#two-way-sync-workflows]
 
-  * The closure of a New Relic issue is triggered when the PagerDuty incident is resolved.
+  Once the destination is saved, a New Relic API key is kept in `api_key_credentials`. The key is sent in a header as part of the callback REST call to New Relic.
```

Comment:
I'd say saved, instead of kept.
Also, in the second sentence, I'd say:
```
The key is sent in a header as part of the REST callback call to New Relic.
```